testingfacade/APIs/schemas/AnswerSchema.json

```diff
@@ -0,0 +1,45 @@
+{
+    "title": "Controller Testing - POST request to testing tool",
+    "description": "Contains answers to a question to be verified by the test suite",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "required": [
+      "name",
+      "answer_response",
+      "time_answered"
+    ],
+    "properties": {
+      "name": {
+        "description": "Question_id sent to Question API",
```

Comment:
Is the answer `name` the same as the question `name` or the question `question_id`?

Reply:
It is supposed to be the question `question_id`. Will change this to make it more clear

Reply:
Call it `question_id` not `name`?